### DISCUSSION TITLE
How to Add ICONS to the navigation bar?

### DISCUSSION THREAD

**Discussion by 971230** [squidfunk/mkdocs-material#7165] [Q&A] [answered]
Hello, I am a fan of mkdocs very much, because my mother tongue is not English, when I consulted the help document, I did not see how to add ICONS in the navigation bar, maybe I was negligent, I found some similar discussions but there is no suitable, I would like to ask how to add it? I'm sorry to interrupt
<img width="831" alt="PixPin_2024-05-08_22-37-17" src="https://github.com/squidfunk/mkdocs-material/assets/37500200/32bb0b45-749b-410c-8990-aa9bad40622a">

**Comment by kamilkrzyskow** [comment]
Hello @971230,
you have to add a header front matter `icon` property to the Markdown file. It's documented here:
- https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-icon

**Reply by 971230** [reply]
I'm glad to hear that. It helps me a lot. But there is still a question, that is, like the effect of the following picture, in the first level of the navigation bar to add ICONS, I would like to ask how to do?
<img width="823" alt="PixPin_2024-05-08_23-13-30" src="https://github.com/squidfunk/mkdocs-material/assets/37500200/0e55a465-9b7a-4a99-bdcf-54c570698022">

**Reply by kamilkrzyskow** [reply, marked as answer]
I'm not sure I understand what the question is. Perhaps you're asking how to add icons to first level sections that don't have pages attached to them. You can't do that. You have to use the approach given above and set the `icon` property in the Markdown file. 
You can connect sections with index pages using `navigation.indexes`:
- https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages

Other similar discussions:
- https://github.com/squidfunk/mkdocs-material/discussions/6419
- https://github.com/squidfunk/mkdocs-material/discussions/6581

**Reply by 971230** [reply]
Wow, great, This document is actually very detailed, I blame myself for not reading it carefully, thank you for your answer, so that my website can look better, thank you, thank you very much💖💖
